components/Captcha.js

```diff
@@ -67,7 +67,8 @@ ReCaptcha.propTypes = {
   onError: PropTypes.func,
 };
 
-const Captcha = ({ onVerify, provider, ...props }) => {
+// eslint-disable-next-line react/display-name
```

Comment:
I think we can do `Captcha.displayName = 'Captcha';` after the component to get rid of the warning